packages/ovh-manager-core/session/session.service.js

```diff
@@ -0,0 +1,31 @@
+export default class SessionService {
+  constructor($q, OvhApiMe) {
+    'ngInject';
+
+    this.$q = $q;
+    this.OvhApiMe = OvhApiMe;
+  }
+
+  getUser(force) {
```

Comment:
Why do we have a parameter `force` ?

Reply:
imho since ovh-api-services already provides caching features we should avoid to perform caching in services
(or stick to one caching solution, either in our angular services or in ovh-api-services)
sometimes it can lead to caching data twice (both in services & api-service library) which is pretty ugly and error prone
what do you guys think?

Reply:
As caching feature is supported in [`ovh-api-services`](https://github.com/ovh-ux/ovh-api-services/blob/master/src/me/me.v6.service.js#L14), `getUser` function should be reduced to : 

```javascript
getUser() {
  return this.OvhAPiMe.v6().get().$promise;
}
```